luaui/Widgets/map_startbox.lua

```diff
@@ -706,8 +661,6 @@ function widget:Update(delta)
 		local newFlipped = getMiniMapFlipped()
 		if flipped ~= newFlipped then
 			flipped = newFlipped
```

Comment:
since you remove the rest of the code at this `if` you can just assign `flipped = getMiniMapFlipped()` and remove the `if`

Reply:
actually looks like `flipped` here only gets used inside the unused method drawFormList, so both could be removed.